ephios/plugins/basesignupflows/flow/coupled.py

```diff
@@ -20,7 +20,13 @@ class CoupledSignupActionValidator(NoSignupSignupActionValidator):
     def get_no_signup_allowed_message(self):
         if self.shift.signup_flow.leader_shift:
             return ActionDisallowedError(
-                _("Participation is coupled to {}.").format(self.shift.signup_flow.leader_shift)
+                mark_safe(
+                    _("Participation is coupled to {}.").format(
+                        f'<a class="link-primary link-underline-opacity-25" '
+                        f'href="#shift-{self.shift.signup_flow.leader_shift.pk}">'
+                        f"{self.shift.signup_flow.leader_shift.get_datetime_display()}</a>"
```

Comment:
maybe include shift title of the leader shift if there is one?

Reply:
yeah just did that